doc/readme/INSTALL.md

```diff
@@ -286,7 +286,7 @@ On a Linux cluster without root access, using [Anaconda](https://www.continuum.i
 
 ## Windows build <a name="manual-windows"></a>
 
-Please see our [AppVeyor](https://ci.appveyor.com/project/vigsterkr/shogun) build.
+Please see our [Azzure Pipelines](https://dev.azure.com/shogunml/shogun/_build?definitionId=2) build.
```

Comment:
`Azure`

Reply:
@gf712 I have fixed the typo!. Please review and le me know if anything has to be changed further